### DISCUSSION TITLE
Difference between ocaml-stdint?

### DISCUSSION THREAD

**Discussion by hirrolot** [yallop/ocaml-integers#47] [General]
There's another library providing fixed-width integer types: [ocaml-stdint]. As someone in need for such functionality, I'm thus asking whether there are significant differences between your library and ocaml-stdint. At first sight, the latter provides some exotic integer types, such as 24, 48, or 56-bit integers, though I don't need them for now. Are there significant implementation differences, and should one library be preferred over another one? Thanks.

[ocaml-stdint]: https://github.com/andrenth/ocaml-stdint

**Comment by yallop** [comment]
There's some discussion here:
https://discuss.ocaml.org/t/idiomatic-ocaml-int8-int16-int32-int64-uint8-uint16-uint32-uint64/11754
